frontend/cypress/tests/chartView/chartView_mergeGroup.cy.js

```diff
@@ -54,4 +54,45 @@ describe('merge group', () => {
       .should('be.visible')
       .should('be.disabled');
   });
+
+  it('merge group contribution bars should have correct width after reload', () => {
+    cy.get('#summary label.merge-group > input:visible')
+      .should('be.visible')
+      .check()
+      .should('be.checked');
+
+    // get the three chart bars and assert they have the correct initial widths
+    cy.get('.summary-chart__contrib--bar')
+      .should('have.length', 3)
+      .then(($bars) => {
+        // calculate the percentage of the width relative to the parent container
+        const parentWidth = $bars.eq(0).parent().width();
+        const width1 = (parseFloat(window.getComputedStyle($bars[0]).width) / parentWidth) * 100;
+        const width2 = (parseFloat(window.getComputedStyle($bars[1]).width) / parentWidth) * 100;
+        const width3 = (parseFloat(window.getComputedStyle($bars[2]).width) / parentWidth) * 100;
+
+        // assert that the widths are close enough to 100% and 50%
+        expect(width1).to.be.closeTo(100, 1);
```

Comment:
I was wondering if instead of hardcoding the values, we could record the values of the number of `.summary-chat__contrib--bar` elements and their respective widths before reloading, then checking if they are the same after reload? So we can avoid hardcoding the values which might potentially change in the future.

Reply:
Great idea, I disliked how hardcoded it was as well, let me modify it to be less hardcoded.